src/azure-cli/azure/cli/command_modules/synapse/manual/operations/artifacts.py

```diff
@@ -465,3 +467,88 @@ def create_or_update_spark_job_definition(cmd, workspace_name, spark_job_definit
     properties = SparkJobDefinition.from_dict(definition_file)
     return sdk_no_wait(no_wait, client.begin_create_or_update_spark_job_definition,
                        spark_job_definition_name, properties, polling=True)
+
+
+def list_sql_scripts(cmd, workspace_name):
+    client = cf_synapse_sql_script(cmd.cli_ctx, workspace_name)
```

Comment:
```suggestion
def list_sql_scripts(cmd, client, workspace_name):
```
I don't think you need `custom_command` since there's no customized logic in `custom.py`
You can just use `g.command('list', 'get_sql_scripts_by_workspace')`

Reply:
When I use g.command('list', 'get_sql_scripts_by_workspace') and remove function list_sql_scripts, there is a error as below:
![image](https://user-images.githubusercontent.com/25128773/144204471-33ba82fc-4240-4fd9-942b-b961d03c92ea.png)